packages/govuk-frontend-review/src/views/component-preview.njk

```diff
@@ -1,5 +1,9 @@
 {% extends "layouts/" + previewLayout | default('layout') + ".njk" %}
 
+{% set templateTitle = (componentName + " " + exampleName | lower) | unslugify %}
```

Comment:
Ah did you definitely want "Accordion default" rather than "Accordion"?

```suggestion
{% set templateTitle = (componentName + (" " + exampleName if exampleName != "default")) | unslugify %}
```

We also don't have `| lower` on the example iframe headings, did you definitely want it?

https://github.com/alphagov/govuk-frontend/blob/545fde1dda58d2b502d3a67618ebd0dc9567ee51/packages/govuk-frontend-review/src/views/macros/showExamples.njk#L10-L16

Reply:
So on the first comment: I'd ideally like some identifier other than the component name because then you've got 2 things with the same title:

1. root component page: {{Component}} - GOV.UK Frontend
2. default example page: {{Component}} - GOV.UK Frontend

Maybe if we checked if it was "default" and then put it at the front instead? So "Default accordion" for the first example and "Accordion blah blah blah" for everything else. What do you think?

Re: the `lower` thing, I need to check again and we no longer need it! I'll remove.

Reply:
You choose, just think it'll be good to match [showExamples.njk](https://github.com/alphagov/govuk-frontend/blob/545fde1dda58d2b502d3a67618ebd0dc9567ee51/packages/govuk-frontend-review/src/views/macros/showExamples.njk#L10-L16) 😊

Reply:
I've implemented it! Let me know what you think. Maybe there's a neater way to format it.